subprojects/core-api/src/main/java/org/gradle/api/file/ProjectLayout.java

```diff
@@ -47,13 +47,24 @@ public interface ProjectLayout extends FileSystemLayout {
 
     /**
      * {@inheritDoc}
+     *
+     *  <p>
+     *  File system locations based on relative paths will be
+     *  resolved against this layout's reference location, as defined by {@link #getProjectDirectory()}.
+     *  </p>
```

Comment:
💅  There seems to be an extra space in front of every javadoc line here

Reply:
✅